src/main/scala/firrtl/passes/Checks.scala

```diff
@@ -414,10 +418,20 @@ object CheckTypes extends Pass {
           errors append new InvalidConnect(info, mname, sx.loc.serialize, sx.expr.serialize)
         case sx: PartialConnect if !bulk_equals(sx.loc.tpe, sx.expr.tpe, Default, Default) =>
           errors append new InvalidConnect(info, mname, sx.loc.serialize, sx.expr.serialize)
-        case sx: DefRegister => sx.tpe match {
-          case AnalogType(w) => errors append new IllegalAnalogDeclaration(info, mname, sx.name)
-          case t if wt(sx.tpe) != wt(sx.init.tpe) => errors append new InvalidRegInit(info, mname)
-          case t =>
+        case sx: DefRegister => {
```

Comment:
Stylistically, we generally don't put brackets around the code following from a case, ie.
```scala
case sx: DefRegister => // we don't usually put { here
```